### PR TITLE
Update plan cache doc

### PR DESCRIPTION
**Pull request by Reminiscent**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add description on the new changes for plan cache.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by qw4990 — APPROVED** [review]
LGTM

File: sql-prepare-plan-cache.md [hunk 1]
```diff
@@ -36,6 +37,7 @@ LRU 链表是设计成 session 级别的缓存，因为 `Prepare` / `Execute` 
 - 当前的 schema 版本，每条执行成功的 DDL 语句会修改 schema 版本；
 - 执行 `Execute` 时的 SQL Mode；
 - 当前设置的时区，即系统变量 `time_zone` 的值；
+- 系统变量 `sql_select_limit` 的值;
```
Comment (TomShawn) [inline sql-prepare-plan-cache.md:40]:
```suggestion
- 系统变量 `sql_select_limit` 的值；
```

File: sql-prepare-plan-cache.md [hunk 2]
```diff
@@ -45,7 +47,8 @@ key 中任何一项变动（如切换数据库，重命名 `Prepare` 语句，
 
 关于执行计划缓存和查询性能有几点值得注意：
 
-- 第一次执行 `Execute` 时（此时还没有缓存计划），会受到已有 SQL Binding 的影响；但是已经缓存的计划，不会受到新创建的 SQL Binding 的影响。同理，已经缓存的计划也不会受到统计信息更新、优化规则和表达式下推黑名单更新的影响。
+- 不管计划是否已经被缓存，都会收到 SQL Binding 的影响。对于没有被缓存的计划，即在第一次执行 `Execute` 时，会受到已有 SQL Binding 的影响；而对于已经缓存的计划，如果有新的 SQL Binding 被创建产生，则原有已经被缓存的计划会失效。
```
Comment (TomShawn) [inline sql-prepare-plan-cache.md:50]:
```suggestion
- 不管计划是否已经被缓存，都会受到 SQL Binding 的影响。对于没有被缓存的计划，即在第一次执行 `Execute` 时，会受到已有 SQL Binding 的影响；而对于已经缓存的计划，如果有新的 SQL Binding 被创建产生，则原有已经被缓存的计划会失效。
```

File: sql-prepare-plan-cache.md [hunk 3]
```diff
@@ -45,7 +47,8 @@ key 中任何一项变动（如切换数据库，重命名 `Prepare` 语句，
...
+- 对于已经被缓存的计划不会受到统计信息更新、优化规则和表达式下推黑名单更新的影响，仍然会使用已经保存在缓存中的计划。
```
Comment (TomShawn) [inline sql-prepare-plan-cache.md:51]:
```suggestion
- 已经被缓存的计划不会受到统计信息更新、优化规则和表达式下推黑名单更新的影响，仍然会使用已经保存在缓存中的计划。
```

**Comment by Reminiscent** [comment]
@TomShawn PTAL

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**